### DISCUSSION TITLE
建议增加框架roadmap

### DISCUSSION THREAD

**Discussion by TrumanDu** [alibaba/ice#6032] [General]
- 了解社区框架的未来发展
- 一些任务也可以分发给更多的外部贡献者
- 我喜欢ice的渐进式胜于umi,但从ice script到ice v3一路的升级工作，感觉越来越大。

**Comment by wssgcg1213** [comment]
收到你的建议, 接下来会做更多开源运作机制相关的事情, 可以关注一下~
